x/deployment/client/cli/tx.go

```diff
@@ -274,7 +274,7 @@ func cmdGroupClose(_ string) *cobra.Command {
 
 	flags.AddTxFlagsToCmd(cmd)
 	AddGroupIDFlags(cmd.Flags())
-	MarkReqGroupIDFlags(cmd, DeploymentIDOptionNoOwner(true))
```

Comment:
`--owner` should be optional so original `MarkReqGroupIDFlags(cmd, DeploymentIDOptionNoOwner(true))` is legit